opentok/opentok.py

```diff
@@ -1122,7 +1132,8 @@ def _create_jwt_auth_header(self):
             "ist": "project",
             "iss": self.api_key,
             "iat": int(time.time()),  # current time in unix time (seconds)
-            "exp": int(time.time()) + (60 * 3),  # 3 minutes in the future (seconds)
+            "exp": int(time.time())
+            + (60 * self._jwt_livetime),  # 3 minutes in the future (seconds)
```

Comment:
Indentation looks weird for this since the operation occurs over multiple lines - suggest adding one more indent to this line.